pages/rotas.vue

```diff
@@ -0,0 +1,115 @@
+<template>
+  <div v-if="cidsRoutes" class="routes-page__container">
+    <div class="routes-page__chips-container">
+      <v-chip
+        v-for="(item, index) in cidsRoutes"
+        :key="index"
+        class="routes-page__chip"
+        close
+        close-icon="mdi-close"
+        :color="cidsRoutes[index].color"
+        text-color="white"
+        @click="()=>updateSelectedCidRoute(index)"
+      >
+        {{index}}
+      </v-chip>
+    </div>
+    <div class="routes-page__steps">
+      <v-carousel v-model="model">
+        <v-carousel-item
+          v-for="(route, index) in cidsRoutes[selectedCidRoute].routes"
+          :key="index"
+        >
+          <v-card class="routes-page__card" height="90%">
+            <v-card-text>
+              <div class="font-weight-bold ml-8 mb-2">Itinerário {{ index + 1 }}</div>
+
+              <v-timeline align-top dense>
+                <v-timeline-item
+                  v-for="(step, indexStep) in route"
+                  :key="indexStep"
+                  :color="cidsRoutes[selectedCidRoute].color"
```

Comment:
```suggestion
                  :color="item.color"
```

Reply:
Acho que o `item` n existe aqui :thinking:

Reply:
Na vdd existe :eita

Reply:
N existe n, ele ta no outro for k k k

Reply:
mas esse for tá dentro dele né, era pra conseguir acessar, chegou a testar?

Reply:
ahhh verdade, nesse for o certo é `route`

Reply:
Nesse caso eu acho que não é também pq o dado no mock ta vindo assim:
`
  "cid01": {
    "routes": [
      [
        {
          "name": "Viaduto Dutra",
          "address": "Avenida 19 entre as ruas 20 x 22.",
          "time": "10: 42am"
        },
        {
          "name": "Viaduto Dutra",
          "address": "Avenida 19 entre as ruas 20 x 22.",
          "time": "10: 42am"
        },
      ]
    ],
    "color": "blue"
  },
`

Ai o `route` vai ser um array dentro de outro array, o `color` vai ser uma prop que fica no objeto pai desse cara